### DISCUSSION TITLE
Dark/Light themes

### DISCUSSION THREAD

**Discussion by havfo** [edumeet/edumeet#837] [Ideas]
Support for choosing between light/dark themes, and save it to users cookie.

**Comment by misi** [comment]
Themes could be configured in app/config. 
We should add example theme for Dark/Light?

**Comment by so010** [comment]
Theming is in place now so just making a dark and light theme is easy now.
Make this a setting and saving in localStorage.
If no setting saved: ask webAPI for default color scheme, see https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
